src/key-handler/index.tsx

```diff
@@ -41,4 +39,34 @@ export function createMatchingKeyHandler(
   };
 }
 
-export { default } from "./body";
+export function KeyHandler(props: KeyHandlerProps) {
```

Comment:
This was "moved" from `body.tsx`, but also refactored to a function component, so it's hard to see what actually changed here.